sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -363,6 +385,7 @@ def _load_from_rest(cls, rest_obj: ComputeResource) -> "ComputeInstance":
             setup_scripts=SetupScripts._from_rest_object(prop.properties.setup_scripts)
             if prop.properties and prop.properties.setup_scripts
             else None,
+            idle_time_before_shutdown=idle_time_before_shutdown,
```

Comment:
Looks like this is handling the response deserialization to ensure the value is round-tripped successfully?

Reply:
Yes, I set it here so that you can manipulate it as a Compute instance object.